paper.tex

```diff
@@ -439,32 +439,77 @@ \subsection*{Edge annotated recombination}
+% Also: are we not defining ARGs as "binary networks"? Might be good to specify this here.
 
 \section*{ARG inference methods}
 Using the terminology developed here to classify ARGs, we review methods developed
-to infer ARGs. We focus our attention on some recent methods and discuss the
+to \emph{infer} ARGs from sequencing data. We focus our attention on some recent methods and discuss the
 properties of the ARGs inferred.
 
-The ARG has been used as a model for recombining DNA sequence data essentially
-since it was first formulated. Early methods focused on finding parsimonious
-ARGs consistent with a given set of
-sequences \citep{hein1990reconstructing, lyngso2005minimum}.
-However, identifying the most parsimonious ARG for a given sequence data set
-is NP-hard \citep{wang2001perfect}, and subsequent methods have resorted to
-various heuristics \citep{fallon2013acg, hein1993heuristic, ignatieva2021kwarg,
-kelleher2019inferring, minichiello2006mapping, mirzaei2017rent,
-parida2008estimating, song2005efficient, thao2019hybrid, zhang2021biobank}.
+The problem of reconstructing ARGs for samples of recombining sequences has been of significant interest
+essentially since the ARG was first defined. Early methods focussed on finding the most 
+\emph{parsimonious} ARGs, i.e.\ those minimising the number of 
+posited recombination events in the history of the sample \citep{hein1990reconstructing, song2003parsimonious,
+lyngso2005minimum}. However, reconstructing a most parsimonious ARG for a given data set
+is NP-hard \citep{wang2001perfect}, and in order to allow for larger input datasets,
+subsequent parsimony-based methods have resorted to various relaxations and heuristics \citep{hein1993heuristic, 
+song2005efficient, wu2008association, camara2016inference, thao2019hybrid, ignatieva2021kwarg}.
 The scalability of these methods varies substantially based on the heuristic in use
-and the underlying data structure, but the most efficient methods can be practically
-used on megabase scale data and tens of thousands of samples under human-like
-parameters \citep{kelleher2019inferring, zhang2021biobank}.
-
-\citep{camara2016inference} is another type of parsimony method.
+and the underlying data structure, with the most efficient tools limited to analysing hundreds of sequences.
+A number of heuristic methods have also been developed which do not (explicitly) focus on reconstructing
+the most parsimonious ARGs, but rather aim for computational efficiency
+while seeking to achieve good accuracy of the inferred genealogies
+\citep{minichiello2006mapping, parida2008estimating, wu2011new, mirzaei2017rent}; some of these methods \citep{kelleher2019inferring, speidel2019method, zhang2021biobank} can be practically
+used on megabase scale data and tens to hundreds of thousands of samples under human-like
+parameters.
+% AI: just to note, these papers do not all agree in their definition of an ARG. 
```

Comment:
All good points! I would be good to mention the disagreement on the definition of an ARG, if you have concrete examples.  Don't worry about getting the narrative right at the moment, just get any information you think is relevant down in sentences.

A more careful classification of the methods might be helpful. As you say, I guess left-to-right vs backwards-forwards in time is one dimension we could split things on.

We should definitely discuss the issue of times-vs-topology. This cuts to the heart of the ARG structure-vs-model issue and we should discuss it carefully. It's probably worth its own section actually.